src/resources/Catalogs/ProductListingConfiguration.ts

```diff
@@ -24,7 +24,11 @@ export default class ProductListingConfiguration extends Resource {
         );
     }
 
-    get(catalogId: string, productListingId: string, productListingConfigurationId: string) {
+    get(
+        catalogId: string,
+        productListingId: string,
+        productListingConfigurationId: string
+    ): Promise<Record<string, ProductListingConfigurationModel>> {
         return this.api.get(
```

Comment:
```suggestion
    get(
        catalogId: string,
        productListingId: string,
        productListingConfigurationId: string
    ) {
        return this.api.get<ProductListingConfigurationModel>(
```

This should work betta

Reply:
This is prettier also, thanks